styles/navigation/nav.css

```diff
@@ -0,0 +1,42 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: white;
+  --secondary-color: rgb(15, 1, 1);
+  --default: rgb(53, 52, 52);
+}
+.nav-container {
+  padding: 1rem;
```

Comment:
suggestion: : You can use global variables for spacing to have consitency throughout the code.

Reply:
definitely, will do that